sources/models/shared-defaults/defaults.toml

```diff
@@ -149,3 +149,11 @@ restart-commands = ["/usr/bin/certdog"]
 
 [metadata.settings.pki]
 affected-services = ["pki"]
+
+# OCI hooks
+
+[settings.oci-hooks]
+log4j-hotpatch-enabled = false
+
+[metadata.settings.oci-hooks]
+affected-services = ["oci-hooks"]
```

Comment:
I'd probably put all this in `shared-defaults/oci-hooks.toml`, because you're going to want a different set of affected services for `shared-defaults/nvidia-oci-hooks.toml` when that shows up.